docs/cases/api/cases-api/cases-api.asciidoc

```diff
@@ -6,8 +6,6 @@ You can create, manage, configure, and send cases to external systems with these
 
 NOTE: Cases return a warning header for deprecated endpoints. The value of the warning header is in the form `299 Kibana-{kibana_version} "{warning_text}"`
 
-* {kibana-ref}/cases-api.html[Cases API]: Used to open and manage security action items.
+* {kibana-ref}/cases-api.html[Cases APIs]: Used to open and manage security action items.
```

Comment:
"Action items" sounds jargony. I'm not sure what a less business-like alternative would be. I'd suggest "cases", but it's still pretty specific to Elastic/Security and it's not all that descriptive so I don't think it's a good alternative.

Reply:
I'll leave it as it is for now but if you think of a better term, that's fine by me!